fdbserver/RestoreApplier.actor.cpp

```diff
@@ -271,6 +271,30 @@ ACTOR Future<Void> applyToDB(Reference<RestoreApplierData> self, Database cx) {
 	}
 
 	state Reference<ReadYourWritesTransaction> tr(new ReadYourWritesTransaction(cx));
+	// Sanity check the restoreApplierKeys, which should be empty at this point
+	loop {
+		try {
+			tr->reset();
+			tr->setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
+			tr->setOption(FDBTransactionOptions::LOCK_AWARE);
+			Key begin = restoreApplierKeyFor(
+			    self->id(), bigEndian64(0)); // Integer must be BigEndian to maintain ordering in lexical order
```

Comment:
It's better to move this comment to the `restoreApplierKeyFor()`, specifying `Version` should be big endian.